numpy/tests/test_public_api.py

```diff
@@ -289,17 +280,17 @@ def is_unexpected(name):
 # These are present in a directory with an __init__.py but cannot be imported
 # code_generators/ isn't installed, but present for an inplace build
 SKIP_LIST = [
-    "numpy.core.code_generators",
-    "numpy.core.code_generators.genapi",
-    "numpy.core.code_generators.generate_umath",
-    "numpy.core.code_generators.ufunc_docstrings",
-    "numpy.core.code_generators.generate_numpy_api",
-    "numpy.core.code_generators.generate_ufunc_api",
-    "numpy.core.code_generators.numpy_api",
-    "numpy.core.code_generators.generate_umath_doc",
-    "numpy.core.code_generators.verify_c_api_version",
-    "numpy.core.cversions",
-    "numpy.core.generate_numpy_api",
+    "numpy._core.code_generators",
+    "numpy._core.code_generators.genapi",
+    "numpy._core.code_generators.generate_umath",
+    "numpy._core.code_generators.ufunc_docstrings",
+    "numpy._core.code_generators.generate_numpy_api",
+    "numpy._core.code_generators.generate_ufunc_api",
+    "numpy._core.code_generators.numpy_api",
+    "numpy._core.code_generators.generate_umath_doc",
+    "numpy._core.code_generators.verify_c_api_version",
+    "numpy._core.cversions",
+    "numpy._core.generate_numpy_api",
 ]
```

Comment:
Are these actually needed anymore now that `_core` is private? I think you might just be able to delete all of this.

Reply:
Right, underscore in the name already makes these modules as skippable.